Patches/HLX Rimworld UNSC Armoury/Weapons_Ranged.xml

```diff
@@ -0,0 +1,402 @@
+            <muzzleFlashScale>9</muzzleFlashScale>
+          </Properties>
+          <AmmoUser>
+            <magazineSize>8</magazineSize>
+            <reloadTime>1.90</reloadTime>
+            <ammoSet>AmmoSet_500SWMagnum</ammoSet>
+          </AmmoUser>
+          <FireModes>
+            <aiAimMode>AimedShot</aiAimMode>
+            <aimedBurstShotCount>1</aimedBurstShotCount>
+          </FireModes>
+          <weaponTags>
+            <li>SimpleGun</li>
+            <li>RUA_Gun</li>
+            <li>CE_OneHandedWeapon</li>
+            <li>CE_Sidearm</li>
+          </weaponTags>
+        </li>
+
+        <!-- == Advanced Assault Rifle == -->
+        <li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+          <defName>RUA_Gun_AdvancedAssaultRifle</defName>
+          <statBases>
+            <WorkToMake>48000</WorkToMake>
+            <SightsEfficiency>1</SightsEfficiency>
+            <ShotSpread>0.10</ShotSpread>
+            <SwayFactor>1.30</SwayFactor>
+            <Bulk>7</Bulk>
+            <Mass>4.32</Mass>
+            <RangedWeapon_Cooldown>0.36</RangedWeapon_Cooldown>
+          </statBases>
+          <Properties>
+            <recoilAmount>1.51</recoilAmount>
+            <verbClass>CombatExtended.Verb_ShootCE</verbClass>
+            <hasStandardCommand>true</hasStandardCommand>
+            <defaultProjectile>Bullet_762x51mmNATO_FMJ</defaultProjectile>
+            <burstShotCount>6</burstShotCount>
+            <ticksBetweenBurstShots>6</ticksBetweenBurstShots>
+            <warmupTime>1.0</warmupTime>
+            <range>48</range>
+            <soundCast>Shot_AssaultRifle</soundCast>
+            <soundCastTail>GunTail_Medium</soundCastTail>
+            <muzzleFlashScale>9</muzzleFlashScale>
+          </Properties>
+          <AmmoUser>
+            <magazineSize>32</magazineSize>
+            <reloadTime>4.5</reloadTime>
+            <ammoSet>AmmoSet_762x51mmNATO</ammoSet>
+          </AmmoUser>
+          <FireModes>
+            <aimedBurstShotCount>3</aimedBurstShotCount>
+            <aiUseBurstMode>true</aiUseBurstMode>
+            <aiAimMode>AimedShot</aiAimMode>
+          </FireModes>
+        </li>
+
+        <!-- == Advanced Battle Rifle == -->
+        <li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+          <defName>RUA_Gun_AdvancedBattleRifle</defName>
+          <statBases>
+            <WorkToMake>44000</WorkToMake>
+            <SightsEfficiency>1.2</SightsEfficiency>
+            <ShotSpread>0.06</ShotSpread>
+            <SwayFactor>1.23</SwayFactor>
+            <Bulk>6</Bulk>
+            <Mass>5</Mass>
+            <RangedWeapon_Cooldown>0.36</RangedWeapon_Cooldown>
+          </statBases>
+          <Properties>
+            <recoilAmount>1.35</recoilAmount>
+            <verbClass>CombatExtended.Verb_ShootCE</verbClass>
+            <hasStandardCommand>true</hasStandardCommand>
+            <defaultProjectile>Bullet_762x54mmR_FMJ</defaultProjectile>
```

Comment:
Why not make this the same caliber as the Assault Rifle?

Reply:
Technically in-lore the Battle Rifle uses **9.4x40mm** as opposed to the AR / DMR that use **7.62x51mm** and honestly i didn't want to make a new caliber so i just slapped this in.

Reply:
Ah, okay. I may got ahead and add a new caliber. :P